README.md

```diff
@@ -9,3 +9,27 @@ Get a list of the PRs that got closed today, and email it to the POs (_this last
 ```
 PO_TOKEN=... bundle exec ruby todays-prs.rb
 ```
+
+
+## 'release-pdf.sh'
+
+# pre-steps
```

Comment:
Nitpicking here. In [Markdow](https://daringfireball.net/projects/markdown/syntax) the number of `#` you use translates to the kind of heading you'll get.

That is, `##` will be `h2`, `#` `h1`. Imho having `## do this` and then `# step` would look weird.

You can see how this would render by selecting the "View" button in the header of this diff

<img width="1123" alt="screen shot 2018-01-25 at 12 48 08 pm" src="https://user-images.githubusercontent.com/1218433/35366513-0aa16292-01ce-11e8-853a-413a89aa9251.png">

Reply:
whoops yeah i thought it was the other way around.